openlibrary/catalog/marc/tests/test_data/bin_expect/710_org_name_in_direct_order.json

```diff
@@ -0,0 +1,41 @@
+{
+  "publish_date": "2004",
+  "publish_country": "cc",
+  "languages": [
+    "chi"
+  ],
+  "title": "\u4e2d\u56fd\u8bd7\u6b4c\u7814\u7a76\u52a8\u6001",
+  "other_titles": [
+    "Zhongguo shi ge yan jiu dong tai",
+    "Newsletter of the Chinese poetry studies"
+  ],
+  "subtitle": "Newsletter of the Chinese poetry studies",
+  "lccn": [
+    "2007255728"
+  ],
+  "oclc_numbers": [
+    "71247531"
+  ],
+  "lc_classifications": [
+    "PL2307 .Z4754"
+  ],
+  "notes": "Latest issue consulted: di 3 ji, published in 2007.",
+  "authors": [
+    {
+      "entity_type": "org",
+      "name": "Shou du shi fan da xue (Beijing, China). Zhongguo shi ge yan jiu zhong xin"
```

Comment:
@tfmorris, thanks for your review and feedback. I started working on all the 7XX / 1XX issues in one PR #9797, but it started getting a bit tangled.

I'm trying to close out some of the basic string improvement issues with this PR for a stable base, and then I'll deal with making original / alternate script handling consistent in the various name fields.

Once this is merged, I'll update the tests to be consistent and make sure the correct script form of author names is where we want it. I think the current behaviour is inconsistent, and sorting it out properly is going to delay some of these more direct fixes. In #9797 I have changed the comma rearranging test expectation to use the native name, but there is still an inconsistency between 1XX and 7XX fields. This PR fixes the comma rearrangement (in the current chosen script). A follow up PR will make that script choice consistent across all name fields.

Reply:
I agree about having the readable UTF-8 in the test expectations too. I'll endeavour to convert to UTF-8 as I touch the various files. I think some of the original test data is generated by Python output, which can display it in either format. I think the current mix is accidental, and UTF-8 is the way to go. 

re. tooling, I do struggle a bit with mixing RTL Hebrew and Arabic string values in otherwise LTR JSON, most tools seem to auto-align or rearrange things in a way that doesn't help. It's the text direction rather than the character sets that causes the problem there.

Reply:
Thanks for the feedback. I wasn't really sure where the dividing lines were, so I erred on the side of over-commenting, figuring that you could just ignore anything that wasn't relevant.